forms/UrlSearchSelect/index.js

```diff
@@ -86,6 +86,7 @@ export default React.createClass({
 
   getParams(query) {
     return merge(this.props.params, {
+      __jsonp: this.props.jsonp,
```

Comment:
Ooo another option is to update getJSON to accept options. Kinda surprised the first time I went to add a param to it and it blew up because it was calling the caching version...